docs/MO_DG/prepare_model/convert_model/Convert_Model_From_MxNet.md

```diff
@@ -22,17 +22,17 @@ MXNet-specific parameters:
   --save_params_from_nd
             Enable saving built parameters file from .nd files
   --legacy_mxnet_model
-            Enable MXNet loader to make a model compatible with the latest MXNet version.
-            Use only if your model was trained with MXNet version lower than 1.0.0
+            Enable Apache MXNet loader to make a model compatible with the latest Apache MXNet version.
+            Use only if your model was trained with Apache MXNet version lower than 1.0.0
   --enable_ssd_gluoncv
             Enable transformation for converting the gluoncv ssd topologies.
             Use only if your topology is one of ssd gluoncv topologies
 ```
 
-> **NOTE**: By default, Model Optimizer does not use the MXNet loader. It transforms the topology to another format which is compatible with the latest
-> version of MXNet. However, the MXNet loader is required for models trained with lower version of MXNet. If your model was trained with an MXNet version lower than 1.0.0, specify the
-> `--legacy_mxnet_model` key to enable the MXNet loader. Note that the loader does not support models with custom layers. In this case, you must manually
-> recompile MXNet with custom layers and install it in your environment.
+> **NOTE**: By default, Model Optimizer does not use the Apache MXNet loader. It transforms the topology to another format which is compatible with the latest
+> version of Apache MXNet. However, the Apache MXNet loader is required for models trained with lower version of Apache MXNet. If your model was trained with an Apache MXNet version lower than 1.0.0, specify the
```

Comment:
Only the first occurrence on the page must be the official name "Apache MXNet". Subsequent ones can be left as MXNet, but using always "Apache MXNet" is also fine.

Reply:
It will be easier to keep track of the corrected pages this way ;)

Reply:
Having this longer name five times in a short sentence does impact readability though :-( (maybe more so if you are familiar with MXNet). I'm fine with leaving it this way, but in general, surely we can search for pages that do include MXNet and do not include "Apache MXNet"? Also I assume this is not a problem, but for these releases lower than 1.0.0 we mention, the product was not called Apache MXNet but just MXNet.

Reply:
Even if MXNet 1.0 was called itself MXNet - it was wrong according to Apache. It should had been called "Apache MXNet (Incubating)"